node/lib/util/submodule_config_util.js

```diff
@@ -71,6 +71,34 @@ const CONFIG_FILE_NAME = "config";
 exports.modulesFileName = ".gitmodules";
 
 /**
+ * Return the relative path from the working directory of a submodule having
+ * the specified `subName` to its .git directory.
+ * @param {String} subName
+ */
+exports.computeRelativeGitDir = function (subName) {
```

Comment:
Why would it change over time? What the config file would tell me? It looks to me that after `git submodule deinit foo` there is no entry in the config file for `foo`.